bazel/cc_toolchains/clang_configuration.bzl

```
@@ -89,6 +89,17 @@ def _compute_mac_os_sysroot(repository_ctx):
     output = _run(repository_ctx, [xcrun, "--show-sdk-path"]).stdout
     return output.splitlines()[0]
 
+def _compute_bsd_sysroot(repository_ctx):
+    """Look around for sysroot. Return root(/) if nothing found."""
+
+    # Try it-just-works for CMake users
```

```suggestion
    # Try it-just-works for CMake users.
```

Style: minor punctuation missing.